modules/hal_nordic/nrfx/Kconfig

```
@@ -573,6 +573,24 @@ config NRFX_UARTE3
 	depends on $(dt_nodelabel_has_compat,uart3,$(DT_COMPAT_NORDIC_NRF_UARTE))
 	select NRFX_UARTE
 
+config NRFX_UARTE_CONFIG_SKIP_GPIO_CONFIG
+	bool "UARTE GPIO configuration support"
+
+config NRFX_UARTE_CONFIG_SKIP_PSEL_CONFIG
+	bool "UARTE PSEL configuration support"
+
+config NRFX_UARTE_CONFIG_TX_LINK
+	bool "UARTE TX transfer linking support"
+
+config NRFX_UARTE_CONFIG_RX_CACHE_ENABLED
+	bool "UARTE RX caching support"
+	depends on $(dt_nodelabel_has_compat,ram3x,$(DT_COMPAT_MMIO_SRAM))
+	default y
```

Shouldn't this be rather:
```suggestion
	default y if $(dt_nodelabel_has_compat,ram3x,$(DT_COMPAT_MMIO_SRAM))
```
?
Is there a reason to disallow this option when there is no such node?